cli/stork-publisher-agent/README.md

```diff
@@ -0,0 +1,63 @@
+# Stork Publisher Agent
+
+The easiest way to become a Stork Publisher is to run the Stork Publisher Agent docker container on your infrastructure and send price updates to the Agent through a local websocket. The Stork Publisher Agent will sign your price updates with your private key and send them to the Stork Network.
+
+## Getting Started
+
+### Setup
+To run the agent on your infrastructure, you'll need to first create a `config.json` file containing all non-secret configuration and a `secrets.json` file containing all secret configuration, and then run the Stork Publisher Agent docker container.
+
+Here we open a websocket at `ws://localhost:5216/publish` which will both EVM sign every update.
+
+`config.json`:
```

Comment:
Is there somewhere we can link to the full list of allowable configurations?

Reply:
good idea - linked to the struct in `config.go`